sdk/servicebus/service-bus/test/internal/receiveAndDeleteMode.spec.ts

```diff
@@ -11,21 +11,21 @@ import {
   ServiceBusMessage,
   ServiceBusReceiver,
   ProcessErrorArgs
-} from "../src";
+} from "../../src";
 
-import { TestClientType, TestMessage, checkWithTimeout } from "./utils/testUtils";
+import { TestClientType, TestMessage, checkWithTimeout } from "../public/utils/testUtils";
 
-import { InvalidOperationInReceiveAndDeleteMode } from "../src/util/errors";
-import { ServiceBusSender } from "../src/sender";
+import { InvalidOperationInReceiveAndDeleteMode } from "../../src/util/errors";
+import { ServiceBusSender } from "../../src";
```

Comment:
Can add this to the first set of imports since both are importing from "../../src"